python/lsst/ts/MTAOS/config_schema.py

```diff
@@ -200,3 +249,123 @@
 additionalProperties: false
 """
 )
+
+WEP_PIPELINE_CONFIG = yaml.safe_load(
+    """
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_MTAOS/blob/master/python/lsst/ts/MTAOS/schema_config.py
+# title must end with one or more spaces followed by the schema version, which
+# must begin with "v"
+title: Wavefront estimation pipeline configuration schema.
+type: object
+
+properties:
+
+  description:
+    type: string
+    description: Description of this pipeline configuration.
+    default: wep basic processing test pipeline
+
+  instrument:
+    type: string
+    description: >-
+      Specify the corresponding instrument for the data we will be using.
+    default: lsst.obs.lsst.LsstComCam
+
+  tasks:
+        type: object
+        default:
+          isr:
+            class: lsst.ip.isr.isrTask.IsrTask
```

Comment:
Why it is `lsst.ip.isr.isrTask.IsrTask` instead of `lsst.ip.isr.IsrTask`?

Reply:
No idea... I just copied it from @jbkalmbach notebooks